### PR TITLE
Enumerate all `documents` configurations

### PR DESCRIPTION
**Pull request by ivanm-gdp**
GitHub won't allow me to create a new branch, so I created a fork.

Found when trying to create a project with custom script and received a MISSING_CUSTOM_SCRIPT error.

### CODE REVIEW

File: create-project-async/src/project.py [hunk 1]
```diff
@@ -50,7 +55,7 @@ def create(base_url, client_id, client_secret, team_id, operations_path, documen
                 job = json_response['data']['launchTextProjectAsync']['job']
                 print(json.dumps(job, indent=1))
                 print('Check job status using command bellow')
-                get_job_status_command = f"python datasaur_api.py get_job_status --base_url {base_url} --client_id {client_id} --client_secret {client_secret} --job_id {job['id']}"
+                get_job_status_command = f"python api_client.py get_job_status --base_url {base_url} --client_id {client_id} --client_secret {client_secret} --job_id {job['id']}"
```
Comment (tonoman3g) [inline create-project-async/src/project.py:58]: kenapa ini direname jd api_client.py @ivanm-gdp ?

File: create-project-async/src/project.py [hunk 2]
```diff
@@ -19,29 +19,34 @@ def create(base_url, client_id, client_secret, team_id, operations_path, documen
...
+                "variables.input.documents." + str(iterator) + ".file"]
             documents.append(document)
-            iterator += 1
 
-        # Use documents.settings and documents.docFileOptions from operations file
-        if (len(operations["variables"]["input"]["documents"]) > 0 and len(documents) > 0):
-            documents[0]["settings"] = operations["variables"]["input"]["documents"][0]["settings"]
-            documents[0]["docFileOptions"] = operations["variables"]["input"]["documents"][0]["docFileOptions"]
+        # do not copy these properties from operations file, we already set them above
+        manual_keys = ["filename", "file"]
```
Comment (tonoman3g) [inline create-project-async/src/project.py:34]: kalau field `name` bagaimana?

Comment (ivanm-gdp) [reply create-project-async/src/project.py:34]: Oh, kalau di code ini, yang diset manual hanya filename + file @tonoman3g 
https://github.com/datasaur-ai/api-client-example/blob/1d1a0b80d88992fa7caa2da77517e32d718cdff4/create-project-async/src/project.py#L22-L31

File: create-project-async/src/project.py (hunk 1 above)
Comment (ivanm-gdp) [reply create-project-async/src/project.py:58]: oh karena nama filenya `api_client.py` @tonoman3g 
https://github.com/datasaur-ai/api-client-example/blob/1d1a0b80d88992fa7caa2da77517e32d718cdff4/create-project-async/api_client.py#L1-L27

File: create-project-async/src/project.py (hunk 2 above)
Comment (tonoman3g) [reply create-project-async/src/project.py:34]: ok

**Pull request merged**